service/controller/clusterapi/v29/adapter/adapter.go

```diff
@@ -48,6 +50,7 @@ type Config struct {
 	StackState                      StackState
 	TenantClusterAccountID          string
 	TenantClusterKMSKeyARN          string
+	TenantClusterAvailabilityZones  []controllercontext.ContextStatusTenantClusterAvailabilityZone
```

Comment:
Is it ok to use this type from `controllercontext` or should this type be duplicated & copied to one in here? I don't think there will ever be import loops in this case so from that point of view this should be fine.